client/src/app/[tenant]/components/useModelToAdvertise.tsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback, useMemo } from 'react';
+import { useLocalStorage } from 'usehooks-ts';
+
+type ModelToAdvertise = {
+  name: string;
+  date: string;
+};
+
+const MODELS_TO_ADVERTISE: ModelToAdvertise[] = [
```

Comment:
Ah nice hardcoding for now. We should make the model data available via an endpoint to make it easier to fetch the release date.